doc/ref_arch/README.md

```diff
@@ -35,3 +35,18 @@ We need to distinguish between architectural principles and architectural requir
 1. **Security compliance:** To ensure the architecture follows the industry best security practices and is at all levels compliant to relevant security regulations.
 
 1. **Resilience and Availability:** To allow High Availability and Resilience for hosted VNFs, and to avoid Single Point of Failure.
+
+<a name="ra-relationship-scope"></a>
+## Reference Architectures Relationships and Scope
+
+<p align="center"><img src="./figures/ref_arch_relationships.png" alt="Scope and Relationships" title="Scope and Relationships" width="80%"/></p>
+<p align="center"><b>Figure 1:</b> Relationships between and scope of the CNTT Reference Architectures</p>
```

Comment:
In my view RA1 and RA2 are independent from each other. It should be possible to build an RA2 compliant solution without having RA1 underneath. For example for a pure RA2 solution there is no need to have OpenStack Nova.

Reply:
That's the intent of the new wording, has that not come across? Or is it the diagram that needs clarity @CsatariGergely ?

Reply:
I certainly agree btw. The idea being that there will be a series of options when it.comes to what RA2 is deployed on. Might be an existing RA1 compliant cloud. Might be a stripped down OpenStack, might be packet.net or AWS. We will need to document any specifics of each platform to guide each related RI. But only if something needs documenting (I.e. if a particular config of the cloud is required in order to meet the requirements of RA2).

Reply:
@CsatariGergely does this diagram clarify it better? (@TamasZsiros does it seem less messy also?)
![image](https://user-images.githubusercontent.com/52716470/68530219-5b9a2980-02fe-11ea-8e94-ab24a9dbccc8.png)

Reply:
> 
> 
> That's the intent of the new wording, has that not come across? Or is it the diagram that needs clarity @CsatariGergely ?

Now this is clear both in the figure and the text. Thanks.

Reply:
But I still think that VM-s running in Kubernetes should not be part of the current relaese of RA1.

Reply:
> But I still think that VM-s running in Kubernetes should not be part of the current relaese of RA1.

I agree - that's a roadmap thing not a scope thing though.  I've raised #573 re. the roadmap.